test/NuGet.Core.Tests/NuGet.ProjectModel.Test/LockFileFormatTests.cs

```diff
@@ -1805,5 +1802,85 @@ public void LockFileFormat_ReadsLockFileWithEmbedAssemblies()
             Assert.Equal("lib/netstandard2.0/packageA.interop.dll", targetLibrary.RuntimeAssemblies.Single().Path);
             Assert.Equal("embed/netstandard2.0/packageA.interop.dll", targetLibrary.EmbedAssemblies.Single().Path);
         }
+
+        [Fact]
+        public void LockFileFormat_WritesFrameworkReference()
+        {
+
+            // Arrange
+            var lockFileContent = @"{
+  ""version"": 3,
```

Comment:
Why is the version 3 and not [4](https://github.com/NuGet/NuGet.Client/pull/2745/files#diff-01bd9ee7e3c773b7d88005bf2daaaf1bR20)?